### PR TITLE
Adding possibility to set environment variables

### PR DESCRIPTION
**Pull request by herschel666**
Hi.

I need a possibility to set environment variables from inside the Gruntfile in an elegant way. I thought the options-object would be a nice place. What do you think?

Best regards!

### CODE REVIEW

**Comment by herschel666** [comment]
Okay, this works on my local machine but unfortunately not on Travis. What would be the best way to deal with this situation? Omitting the test? Omitting the whole feature?? …

File: tasks/php.js [hunk 1]
```diff
@@ -60,6 +61,7 @@ module.exports = function (grunt) {
 			args.push(options.router);
 		}
 
+		grunt.util._.extend(process.env, options.env);
```
Comment (sindresorhus) [inline tasks/php.js]: This is wrong. Just pass `options.env` to the `env` option in `spawn`.

Comment (herschel666) [inline tasks/php.js]: Still not passing. Sorry for the hazzle!

**Comment by sindresorhus** [comment]
@herschel666 Can you fix the merge conflict?

Regarding the travis failure, just ignore it ;)

**Comment by kevva** [comment]
Could you also squash the commits?

**Comment by sindresorhus** [comment]
Thanks @herschel666 :)

**Pull request closed without merging**